bindings/nodejs/lib/types/block/output/feature.ts

```diff
@@ -113,9 +114,9 @@ class BlockIssuerFeature extends Feature {
     /**
      * The Block Issuer Keys.
      */
-    readonly publicKeys: Set<HexEncodedString>;
+    readonly publicKeys: Set<Ed25519BlockIssuerKey>;
```

Comment:
Should be blockIssuerKeys

https://github.com/iotaledger/tips-draft/pull/24/files#diff-04aff9a5004dd8023df5735c866f69254359635764a00e9d9215500a9676319bR2267